### DISCUSSION TITLE
Feeds not working

### DISCUSSION THREAD

**Discussion by trekzavier** [spacecowboy/Feeder#290] [General]
Hello, I been a user of spacecowboys reader for some time and really like it. But for some reason the reader is blocking open rss feeds. I really would like to have these feeds work in spacecowboy. I emailed open rss support and they say that spacecowboys is getting blocked because it bringing down their server and point me to issue https://openrss.org/issue/136 can anyone help with this?

**Comment by spacecowboy** [comment]
I can have a look at what responses they are returning, and if feeder is handling them correctly or not.

and byw, the name is Feeder. "spacecowboys rss reader" is just the useragent

**Reply by markcellus** [reply]
Hey, I'm an engineer for Open RSS and just found this issue. Thanks @trekzavier for filing it! 

@spacecowboy I believe you filled out the contact form on the website. We replied to the email address entered, but our email reply returned as undeliverable. Can you shoot me an email directly at mark [at] openrss.org? Thanks!

**Reply by spacecowboy** [reply]
OK, sent @markcellus !

I must have typed it wrong or something

**Comment by spacecowboy** [comment]
Fix in https://github.com/spacecowboy/Feeder/pull/292

**Reply by trekzavier** [reply]
ok thank you very much for this one :) do we need that to fix it?

**Reply by spacecowboy** [reply]
Yes, or you will just keep getting blocked.

Having multiple openrss feeds is the issue. The more feeds you have, the more likely it is you get banned.

But with this fix, you'll just get a temporary pause in syncing.

**Reply by trekzavier** [reply]
Ok how long is the pause? I use a lot of feeds how many feeds can I have?

**Reply by spacecowboy** [reply]
The pause is dictated by openrss.org. They send a value saying that feeder should wait for X seconds. It starts at like 1-2 minutes.

You can also subscribe to the feeds directly - instead of going through openrss. That would remove the entire limitation.

**Reply by spacecowboy** [reply]
@trekzavier please update to latest 2.6.25. It should resolve some of your issues

**Reply by trekzavier** [reply]
thank you! usually I subscribe to rss feeds directly but open rss is great at rss feeds for websites without them. so I prefer to use open rss for that. 1 or 2 minutes is fine. that's no "pause", that is just how it should work normally haha
